rmw_fastrtps_cpp/src/rmw_count.cpp

```diff
@@ -48,34 +48,19 @@ rmw_count_publishers(
 
   CustomParticipantInfo * impl = static_cast<CustomParticipantInfo *>(node->data);
 
-  std::map<std::string, std::vector<std::string>> unfiltered_topics;
```

Comment:
I think you need to set `*count` to 0 here (like you do in `rmw_count_subscribers`), no?

Reply:
Yes, I'll fix this.

Reply:
PR Updated